sample-apps/spark/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM gradle:6.5.0-jdk11 as builder
```

Comment:
Though then I noticed we have this Dockerfile - do we need the dockerfiles?

Reply:
We do need dockerfiles for sample apps. Since we are planning to build docker images and publish to docker hubs for other repos to consume.